src/config.py

```diff
@@ -17,6 +17,7 @@
 
 def load_config(path="./config.toml"):
     """Loads the config from `path`"""
+    print("loaded config")
```

Comment:
Wait, what?. First of all you print this `BEFORE` its `opened`, checked if it exists.
```suggestion
    try:
        print("Looking for config...")
        # Open config. etc... etc....
    except Exception as e:
        logging.warn(f'Error: {e}')
```
No exception checking no nothing.